src/main/java/ch/uzh/ifi/hase/soprafs21/entity/Match.java

```diff
@@ -0,0 +1,47 @@
+/*package ch.uzh.ifi.hase.soprafs21.entity;
```

Comment:
Match entity already exist. Please pull master first, and  merge into personal branch

Reply:
aight